content/publishing/email-and-slack-notifications.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Receive artifacts of successful builds and build status updates with email or Slack.
+title: Email and Slack notifications
+weight: 8
+---
+
+## Email
+
+Email publishing settings can be found in **App settings > Publish > Email**.
+
+Email publishing is the only publishing option which is set enabled (`Enable email publishing`) by default. Default email from the service used to log in (Github, Bitbucket, Gitlab) is used. You can fill in multiple emails.
+
+If a build has finished successfully, release notes (if passed) and generated artifacts are published to the provided email.
+
+If a buid failed, failure reason will be sent with the link to check build logs. Check `Publish artifacts even if tests fail` to publish artifacts even when one or more tests fail. Leaving this option unchecked will attach generated atrifacts (if any) only when build was successful will publish only successful.
```

Comment:
_Leaving this option unchecked will attach generated atrifacts (if any) only when build was successful will publish only successful._ needs rephrasing

Reply:
tried to rephrase in d11b88d